config/explorer-testnet-config.yaml

```diff
@@ -0,0 +1,104 @@
+# Explorer DB Node configuration
```

Comment:
Just a question.Is there any reason why we do this instead of doing a script that generates it from mainnet config like we do with `scripts/gen-tx-submit-config.sh`.

Reply:
No, just noticed your message about it on Slack. We can change it.

Reply:
See what Duncan says, but the script is shorter and only changes the things that must  be changed.

Reply:
I think it's useful for 3rd parties to have concrete examples they can us, rather than scripts that generate things. It means our README instructions can be as simple as "replace mainnet with testnet in all the cases below".